serverless.yml

```diff
@@ -171,6 +185,58 @@ resources:
               Principal:
                 AWS:
                   - !GetAtt [IamRoleLambdaExecution, Arn]
+
+    # Create S3 Bucket Policy which permits access only from AWS Lambda and AWS CloudFront
+    S3BucketHttpsPublicBucketPolicy: 
+      Type: AWS::S3::BucketPolicy
+      Condition: HttpsUi
+      Properties: 
+        Bucket: 
+          !Ref S3BucketPublicBucket
+        PolicyDocument: 
+          Statement:
+            - 
+              Action: 
+                - "s3:GetObject"
+              Effect: "Allow"
+              Resource: "arn:aws:s3:::${self:custom.config.public_bucket_name}/*"
+              Principal:
+                AWS: !Join [" ", ["arn:aws:iam::cloudfront:user/CloudFront Origin Access Identity", !Ref EphemeraCloudfrontOriginAccessIdentity]]
```

Comment:
Will this work @DaveOsment ? Won't it just have two ARNs separated by a space? is "CloudFront Origin Access Identity" even valid? I haven't use this functionality before.

Reply:
When AWS auto-generated this for me, I asked exactly the same question myself and did a double-take! 

Upon testing, it creates the following entry in the S3 Bucket Policy:

```"Principal": {
                "AWS": "arn:aws:iam::cloudfront:user/CloudFront Origin Access Identity <Redacted>"
            },````

This is permitting me to access Ephemera via CloudFront under test.